### PR TITLE
[LANGUAGE] improved error productions for at

### PR DESCRIPTION
**Pull request by akseron**
Changes in error production in level 4 lark file and check_var_usage in main hedy file to support better syntax error messages for the 'at' command. 

Fixes https://github.com/hedyorg/hedy/issues/4780

### CODE REVIEW

File: translations/en/LC_MESSAGES/messages.po
Comment (Felienne) [inline translations/en/LC_MESSAGES/messages.po:148]: Hi @akseron!

It is not good practice to add to the po files directly, that will create problems. Instead, you have to add the new exception here: 

https://github.com/hedyorg/hedy/blob/5ba17c5c738ecb015e68b2e05dd77c5429dd6d0c/content/error-messages.txt#L10

And then run the extract script (see wiki) and the strings will be entered into the pot and po files automatically.

Comment (Felienne) [reply translations/en/LC_MESSAGES/messages.po:148]: See here: https://github.com/hedyorg/hedy/wiki/Hedy-Development-Process#adding-new-translation-keys

File: exceptions.py
Comment (Felienne) [inline exceptions.py:237]: Hi @akseron!

Is this something you want to do in this PR or can we merge this now and do this later on?

In any case, can you add a date and your initials to the TODO so we know who to ask in the future? :)

Comment (akseron) [reply exceptions.py:237]: Oh I had forgotten to remove that, that was part of an earlier part of the code adjustment.
Will keep in mind to do so!

Comment (Felienne) [reply exceptions.py:237]: Does that mean it can be removed? If so, please do :) 

I have fixed the po issues, so I think.... we might be ready to merge?!

Comment (akseron) [reply exceptions.py]: Hahaha, I did want to ask about one more thing before that. In the testing in tests/test_level/test_level_04.py I had changed the parameter for skipping faulty code, however I'm not 100% sure that I did it in the right place. If that is correct it should be ready to merge :D

File: tests/test_level/test_level_04.py
Comment (Felienne) [inline tests/test_level/test_level_04.py:582]: This is correct actually, it disables a special setting we have for skipping error lines. The only thing is that I prefer False for false over 0.

File: tests/test_level/test_level_04.py
Comment (Felienne) [inline tests/test_level/test_level_04.py:583]:
```suggestion
            skip_faulty=False,
```

**Comment by Felienne** [comment]
Hi @akseron!

This can be merged if you update the PR description (the first message here) to describe briefly what you did and why (and whether there is an open issue related to this PR)

File: hedy.py
Comment (Felienne) [inline hedy.py:734]: I think we want to remove all these debugging prints before merging

**Review by Felienne — CHANGES_REQUESTED** [review]

File: tests/test_level/test_level_04.py
Comment (Felienne) [inline tests/test_level/test_level_04.py:578]: This should be "ad" I think? Not sure why this test was not failing, it should (since the error is not raised). I do get an error locally so maybe something is up with the tests?

File: tests/test_level/test_level_04.py
Comment (Felienne) [inline tests/test_level/test_level_04.py:577]:
```suggestion
        dieren is Hond, Kat, Kangoeroe, Kaketoe
```

**Review by Felienne — APPROVED** [review]
I removed the original comment documenting this issue from over 2 years ago:

image

With that, we are good to go!!

**Comment by Felienne** [comment]
And congrats on getting your first contribution accepted @akseron! It was quite a ride, but you did it!!

**Pull request merged**
